perfrunbook/debug_hw_perf.md

```diff
@@ -136,3 +136,44 @@ Backend stalls are caused when the CPU is unable to make forward progress execut
 6. If back-end stalls due to the cache-system and memory system are the problem, the data-set size and layout needs to be optimized.
 7. Proceed to [Section 6](./optimization_recommendation.md) to view optimization recommendations for working with a large data-set causing backend stalls.
 
+### Capturing Coherent Mesh Network hardware event counters
+
+The CMN connects the CPUs to each other, to the memory controller, the I/O subsystem and provides System Level Cache.
+Its PMU counts events such as requests to SLC, DRAM (memory bandwidth), IO bus requests or coherence snoop events.
+These metrics can be used to assess an application's utilization of such system level resources and if resources are used efficiently.
+CMN counters are only accessible on metal-type instances and certain OSes and kernels.
+
+
+|Distro      |Kernel   | Graviton2 (c6g) | Graviton3 (c7g) |
+|------------|---------|-----------------|-----------------|
+|Ubuntu-20.04| 5.15    |  yes            |    no           |
+|Ubuntu-20.04| >=5.19  |  yes            |    yes          |
+|Ubuntu-22.04| 5.15    |  no             |    no           |
+|Ubuntu-22.04| >=5.19  |  yes            |    yes          |
+|AL2         | 5.10    |  no             |    no           |
```

Comment:
5.15 kernel?

Reply:
Ubuntu-22.04 with 5.15 practically twists your arm to update to 5.19.
You can force support by explicitly installing the 5.15 extra modules:
sudo apt install linux-modules-extra-5.15.0-1019-aws
After the next reboot you likely run 5.19, since it gets installed by apt update && apt install linux-modules-extra-aws.
Go into that level of detail for 5.15 on Ubuntu-22.04?

Reply:
i just make 5.19 default on Ubuntu-22 and strike the 5.15 line for it.